### DISCUSSION TITLE
Create record and changes the Item affected

### DISCUSSION THREAD

**Discussion by tonyhart7** [pocketbase/pocketbase#329] [Q&A]
I want to do like 
create record with certain amount and reduce the amount from item that affected
example: balance in account profile

**Comment by ganigeorgiev** [comment]
Currently you'll have to send 2 consequent requests to update the related record manually OR use the [Event hooks](https://pocketbase.io/docs/event-hooks/).

In future versions you will be able to resolve this from the Admin UI via:
- computed fields - https://github.com/pocketbase/pocketbase/issues/311
- collection triggers - https://github.com/pocketbase/pocketbase/issues/317
